Library/ViewModels/NoShippingPledgeViewModelTests.swift

```diff
@@ -5226,4 +5229,50 @@ final class NoShippingPledgeViewModelTests: TestCase {
       )
     }
   }
+  
+  func testShowPledgeOverTimeUI_True() {
+    withEnvironment(currentUser: nil) {
```

Comment:
nit: I don't think we need to pass in currentUser here. It'll be `nil` by default and it doesn't look like we're referencing it anywhere in this test anyway.